src/legacy/ui/public/agg_types/buckets/histogram.js

```diff
@@ -76,13 +76,13 @@ export const histogramBucketAgg = new BucketAggType({
     {
       name: 'interval',
       editorComponent: NumberIntervalParamEditor,
-      modifyAggConfigOnSearchRequestStart(aggConfig, searchSource) {
+      modifyAggConfigOnSearchRequestStart(aggConfig, searchSource, searchRequest) {
```

Comment:
can we rather pass abortSignal in here to have same implementation as in other places ?( abortSignal.addEventListener)